debian8/templates/csv/services_disabled.csv

```diff
@@ -1,2 +1,2 @@
-# service_name, package_name, daemon_name (as recognized by chkconfig / systemd. To be used when daemon_name differs from service_name)
+# <service_name, package_name, daemon_name[, dont_mask_service]> (as recognized by chkconfig / systemd. To be used when daemon_name differs from service_name) (if dont_mask_service is set the service will not be masked, can be any string)
```

Comment:
This was my first proposal if you see here: https://github.com/ComplianceAsCode/content/pull/4633/commits/295b88a0ee80d943eeca16b26a5d2e40f9c4162b#diff-2949e8573ac963346b7b2c124319d617R1

But I was not sure how to handle the default value and that would require to change all service_disabled.csv files. It makes sense to have `mask_service` instead of this. I'll work on that.